src/color/Color.ts

```diff
@@ -93,13 +112,15 @@ type ColorSourceTypedArray = Float32Array | Uint8Array | Uint8ClampedArray;
  * new Color({ h: 0, s: 100, l: 50, a: 0.5 }).toArray(); // [1, 0, 0, 0.5]
  * new Color('hsl(0, 100%, 50%, 50%)').toArray(); // [1, 0, 0, 0.5]
  * new Color({ h: 0, s: 100, v: 100, a: 0.5 }).toArray(); // [1, 0, 0, 0.5]
+ * ```
  * @since 7.2.0
  * @memberof color
  */
 export class Color
 {
     /**
      * Default Color object for static uses
+     * @ignore
```

Comment:
this shouldn't be ignored. `Color.shared` is definitely a public API and useful for just converting colors without creating a new object

also why are all these PR's being merged without review?